platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -252,7 +274,22 @@ namespace platoon_strategic_ihp
             */
             void run_candidate_leader();
 
-        
+            // -------------- UCLA: add two states for cut-in join ------------------
```

Comment:
Above, on lines 241 & 264, member variables pm_ and pose_ecef_point_ are declared as public.  As a rule, these should be in the private section.  If they really need to be public, then there should be a comment explaining why.